subsys/tfm/boards/nrf5340_cpuapp_eng_a/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2021, Nordic Semiconductor ASA.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+#-------------------------------------------------------------------------------
+
+cmake_policy(SET CMP0076 NEW)
+set(CMAKE_CURRENT_SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR})
+set(NRF_BOARD_SELECTED True)
+
+add_subdirectory(../../../../../modules/tee/tfm/trusted-firmware-m/platform/ext/target/nordic_nrf/common/nrf5340 nrf5340)
```

Comment:
We should do something with this. my eyes are hurting me.

Reply:
how about:
```suggestion
add_subdirectory(${Trusted\ Firmware\ M_SOURCE_DIR}/platform/ext/target/nordic_nrf/common/nrf5340 nrf5340)
```

Reply:
Done